worldedit-core/src/main/java/com/sk89q/worldedit/EditSession.java

```diff
@@ -1755,8 +1846,8 @@ public int stackCuboidRegion(
      * Move the blocks in a region a certain direction.
      *
      * @param region       the region to move
-     * @param dir          the direction
-     * @param distance     the distance to move
+     * @param offset the offset
+     * @param multiplier the number to multiply the offset by
```

Comment:
Again this is much more a direction than it is an offset. The move command uses "offset" as a unity vector so it does just represent a direction. If you want to make it clear that it can also be an "offset", the directional nature of this should not be ignored.